### ISSUE TITLE
Checkbox component does not expose aria-controls property

### ISSUE THREAD

**Issue by stephpic** [OfficeDev/office-ui-fabric-js#362]
Checkbox component does not expose aria-controls property.  Ideally, should be a way for a checkbox change event to cause a screen reader to announce that another component has been updated.  

In this specific case, we are trying to use a checkbox list to filter a search results list.  Checking or unchecking a filter will cause the search results to update.

**Comment by Linda-Editor** [comment]
Microsoft no longer supports this content and will not be responding to bugs or issues. We recommend that you use the newer version, [Office UI Fabric](https://developer.microsoft.com/fabric), with React as your front-end framework. We are closing this issue; if you still need assistance with Fabric.js, visit [Stack Overflow/office-ui-fabric](https://stackoverflow.com/questions/tagged/office-ui-fabric).
